handlers/user.go

```diff
@@ -190,8 +191,24 @@ func (h *handler) UserInfo(w http.ResponseWriter, r *http.Request) {
 		handleError(w, http.StatusInternalServerError, log.ErrorLevel, msg, err, InternalServerError)
 		return
 	}
+	var syncMode SyncMode
+	if usr.AutoSync {
+		syncMode = Auto
+	} else if usr.SpotifySynchronized {
+		syncMode = ForceSync
+	} else {
+		syncMode = ForceDesync
+	}
 
-	jsonResponse(w, usr)
+	response := &struct {
+		UserInfo *user.Model `json:"user_info"`
+		SyncMode SyncMode    `json:"sync_mode"`
+	}{
+		UserInfo: usr,
+		SyncMode: syncMode,
+	}
```

Comment:
you could embed syncMode and UserInfo in the new struct
https://golang.org/doc/effective_go.html#embedding